wgsl/index.bs

```diff
@@ -4339,9 +4336,10 @@ access mode; the default is read.
 A <dfn>texture resource</dfn> is a variable whose [=effective-value-type=] is a [=texture type=].
 It is declared at [=module scope=].
 It holds an opaque handle which is used to access the underlying grid of [=texels=] in a [=texture=].
-The handle itself is in the [=address spaces/handle=] address space and is is always read-only.
-In many cases the underlying texels are read-only.
-For a write-only [=storage texture=], the underlying texels are write-only.
+The handle itself is in the [=address spaces/handle=] address space and is always read-only.
+In many cases the underlying texels are read-only, and we say the texture variable immutable.
+For a write-only [=storage texture=], the underlying texels are write-only, and by convention
```

Comment:
Should "by convention" be trailed by comma to make it a comma-set parenthetical expression?

Reply:
It can be either way.
Adding the extra comma slows the reader down, which can disrupt the thought process. That's why I left it out.

In fact, you could argue that a comma could both precede and follow "by convention"

> For a write-only storage texture, the underlying texels are write-only, and, by convention, we say the texture variable is mutable.

The commas surround the text that could be deleted. That is, we could have written:

> For a write-only storage texture, the underlying texels are write-only, and we say the texture variable is mutable.

I searched  for an example using "by convention" and found this one that doesn't use commas

https://ell.stackexchange.com/questions/129446/what-does-by-convention-mean 

> The two are independent but by convention campaign together in federal elections and form one group in the Bundestag.

Reply:
@dneto0 Right, it might have been my misunderstanding about the punctuation of left-dislocations. Thanks!